### PR TITLE
The Authentication Backends are responsible for permissions handling.

### PR DESCRIPTION
**Pull request by Rayco**
The Authentication Backends are responsible for permissions handling.
Make sure the CASBackend extends of ModelBackend.

Out[2]: user.get_all_permissions()
Out[2]: set()

After:

Out[2]: user.get_all_permissions()
Out[2]: {u'my_app.add_mymodel',
 u'my_app.change_mymodel',
 u'my_app.delete_mymodel'}

### CODE REVIEW

File: django_cas_ng/backends.py [hunk 1]
```diff
@@ -232,7 +233,7 @@ def _verify_cas2_saml(ticket, service):
 _verify = _PROTOCOLS[settings.CAS_VERSION]
 
 
-class CASBackend(object):
+class CASBackend(ModelBackend, object):
```
Comment (pipermerriam) [inline django_cas_ng/backends.py]: Maybe take `object` out of the inheritance because it's the base for `ModelBackend`?

**Comment by mingchen** [comment]
Thank you.

**Comment by Rayco** [comment]
You're welcome!

Could you add me to credits? Thanks.

**Comment by mingchen** [comment]
Sure.

File: django_cas_ng/backends.py (hunk 1 above)
Comment (mingchen) [inline django_cas_ng/backends.py]: make sense.

**Pull request merged**